server/storage/interface.go

```diff
@@ -39,4 +39,15 @@ type MetaStore interface {
 	// Delete removes all metadata for a given GUN.  It does not return an
 	// error if no metadata exists for the given GUN.
 	Delete(gun string) error
+
+	// GetChanges returns an ordered slice of changes. It starts from
+	// the change matching changeID, but excludes this change from the results
+	// on the assumption that if a user provides an ID, they've seen that change.
+	// If changeID is 0, it starts from the
```

Comment:
IIRC, we discussed not having a "latest" indicator - but a change ID can be empty for "the beginning"?  I wasn't sure if that was going to be the eventual goal or if it would apply to the current changes - if so, we should probably modify the comment.